batchflow/models/tf/layers/drop_block.py

```diff
@@ -32,11 +31,21 @@ def dropblock(inputs, dropout_rate, block_size, is_training, data_format, global
         first positional argument.
     seed : int
         Seed to use in tf.distributions.Bernoulli.sample method.
-
-    Returns
-    -------
-    tf.Tensor
     """
+    def __init__(self, dropout_rate, block_size, data_format, global_step=None, seed=None, **kwargs):
+        self.dropout_rate, self.block_size = dropout_rate, block_size
+        self.data_format, self.global_step, self.seed = data_format, global_step, seed
+        self.kwargs = kwargs
+
+    def __call__(self, inputs, training):
+        return dropblock(inputs, dropout_rate=self.dropout_rate, block_size=self.block_size,
+                         is_training=training, data_format=self.data_format,
+                         global_step=self.global_step, seed=self.seed, **self.kwargs)
+
+
+
+def dropblock(inputs, dropout_rate, block_size, is_training, data_format, global_step=None, seed=None, **kwargs):
```

Comment:
Do you need it as separate function? If yes, why not `@add_as_function`?

Reply:
I can move it inside `__call__`, but that can wait for other things to be polished